src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -85,7 +85,8 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
 
         foreach ($lines as &$line) {
             $line = sprintf('%s%s', $prefix, $line);
-            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line));
+            //https://github.com/symfony/symfony/pull/14600
+            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line) + substr_count($line, '\<'));
```

Comment:
Actually, it's a good catch, I didn't notice this method. :smiley: 
But, is anything else than this [line](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Formatter/OutputFormatter.php#L182) useful in there ? (The method is called in `Helper::strlenWithoutDecoration`).

Does it justify the overhead if it isn't the case ?